model/src/main/kotlin/utils/OrtResultExtensions.kt

```diff
@@ -42,6 +43,18 @@ fun OrtResult.addPackageConfigurations(packageConfigurationProvider: PackageConf
     return copy(resolvedConfiguration = resolvedConfiguration.copy(packageConfigurations = packageConfigurations))
 }
 
+/**
+ * Add all package curations from [packageCurationProviders] that match any packages in this [OrtResult] to
+ * [OrtResult.resolvedConfiguration], overwriting any previously contained package curations. The
+ * [packageCurationProviders] must be ordered highest-priority-first.
+ */
+fun OrtResult.addPackageCurations(packageCurationProviders: List<Pair<String, PackageCurationProvider>>): OrtResult {
```

Comment:
Would it make sense to name this `setPackageCurations()`, because "adding" might hide the fact that existing curations get dropped.

Reply:
Yes, I will rename them in a follow-up for that reason.